### PR TITLE
Added SwiftLint as pod dependency

### PR DESCRIPTION
**Pull request by LeandroHiga**
#### Description:

* Add SwiftLint as pod dependency

---
#### Tasks:

* [ISSUE-176](https://github.com/rootstrap/ios-base/issues/176)

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ target 'ios-base' do
   pod 'Fabric', '~> 1.9.0'
   pod 'Crashlytics', '~> 3.12.0'
   pod 'R.swift', '~> 5.0.3'
+  pod 'SwiftLint'
```
Comment (pMalvasio) [inline Podfile:13]: Follow the same notation and after running the `pod install` command, specify the current version `, '~> x.x.x'`

File: Podfile.lock [hunk 2]
```diff
@@ -126,7 +129,8 @@ SPEC CHECKSUMS:
   R.swift.Library: cfe85d569d9bae6cb262922db130e7c3a7a5fad1
   RSFontSizes: cf14ae41c2807b66573f7064528ccff8c98251c9
   Swifter: 2327ef5d872c638aebab79646ce494af508b0c8f
+  SwiftLint: 99f82d07b837b942dd563c668de129a03fc3fb52
 
-PODFILE CHECKSUM: 4f3f540230adcc23bc2c88e628e6851c38a7c3b0
+PODFILE CHECKSUM: dc61e6efb399e5661257f09af3c799e7f8af1085
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.11.0.beta.1
```
Comment (pMalvasio) [inline Podfile.lock:136]: Watch out with the `cocoapod` version 🙈 I'd recommend `1.11` to keep things stable. Maybe anyone else have another opinion about the version 🤔

**Review by pMalvasio — APPROVED** [review]

**Review by germanStabile — APPROVED** [review]

**Review by CamilaMoscatelli — APPROVED** [review]

File: ios-base.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -805,7 +805,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\nswiftlint\nelse\necho \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
+			shellScript = "if which \"${PODS_ROOT}/SwiftLint/swiftlint\" >/dev/null; then\n${PODS_ROOT}/SwiftLint/swiftlint\nelse\necho \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```
Comment (glm4) [inline ios-base.xcodeproj/project.pbxproj:808]: The entire script should be replaced by:
"${PODS_ROOT}/SwiftLint/swiftlint"

**Review by glm4 — CHANGES_REQUESTED** [review]

File: ios-base.xcodeproj/project.pbxproj (hunk 3 above)
Comment (glm4) [reply ios-base.xcodeproj/project.pbxproj:808]: @LeandroHiga ☝️

**Review by glm4 — APPROVED** [review]

**Pull request merged**